tests/unittests/api/utils_post_data.py

```diff
@@ -34,6 +34,7 @@
         "year": 2016,
         "logo": ""
     },
+    'licence_details': {},
```

Comment:
I don't think this needs to be there in POST API schema as it is an auto-generated field.